tests/components/transmission/test_config_flow.py

```diff
@@ -113,6 +116,33 @@ def init_config_flow(hass):
     assert result["data"]["options"][CONF_SCAN_INTERVAL] == DEFAULT_SCAN_INTERVAL
 
 
+async def test_options(hass):
+    """Test updating options."""
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        title=CONF_NAME,
+        data={
+            "name": DEFAULT_NAME,
+            "host": HOST,
+            "username": USERNAME,
+            "password": PASSWORD,
+            "port": DEFAULT_PORT,
+            "options": {CONF_SCAN_INTERVAL: DEFAULT_SCAN_INTERVAL},
+        },
+        options={CONF_SCAN_INTERVAL: DEFAULT_SCAN_INTERVAL},
+    )
+
+    flow = config_flow.TransmissionOptionsFlowHandler(entry)
```

Comment:
You can use the static method of the config flow to get the options flow.

Reply:
Done